api/tests/integration/AuthTest.php

```diff
@@ -106,6 +91,7 @@ public function testLoginUserMissCredentials()
 
     public function testFailedTokenEncoding()
     {
+        $this->markTestSkipped('why check that?');
```

Comment:
code coverage of what? =)
it's namshi's responsibility to cover that case, not our app